var/spack/repos/builtin/packages/py-chex/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyChex(PythonPackage):
+    """Chex is a library of utilities for helping to write reliable JAX code."""
+
+    homepage = "https://github.com/deepmind/chex"
+    url = "https://pypi.io/packages/source/c/chex/chex-0.1.0.tar.gz"
+
+    version('0.1.0', sha256='9e032058f5fed2fc1d5e9bf8e12ece5910cf6a478c12d402b6d30984695f2161')
+    depends_on('py-absl-py@0.9.0:', type=('build', 'run'))
+    depends_on('py-dm-tree@0.1.5:', type=('build', 'run'))
+    depends_on('py-jax@0.1.55:', type=('build', 'run'))
+    depends_on('py-jaxlib@0.1.37:', type=('build', 'run'))
+    depends_on('py-numpy@1.18.0:', type=('build', 'run'))
+    depends_on('py-toolz@0.9.0:', type=('build', 'run'))
+    depends_on('py-dataclasses', type=('build', 'run'), when='^python@3.6.00:3.6')
```

Comment:
```suggestion
    depends_on('py-dataclasses@0.7:', type=('build', 'run'), when='^python@3.6')
```

Reply:
removed the dep outright as we're doing python 3.7+